var/spack/repos/builtin/packages/hipfft/package.py

```diff
@@ -16,13 +16,14 @@ class Hipfft(CMakePackage):
 
     homepage = "https://github.com/ROCmSoftwarePlatform/hipFFT"
     git      = "https://github.com/ROCmSoftwarePlatform/hipFFT.git"
-    url      = "https://github.com/ROCmSoftwarePlatform/hipfft/archive/rocm-5.1.3.tar.gz"
+    url      = "https://github.com/ROCmSoftwarePlatform/hipfft/archive/rocm-5.2.0.tar.gz"
     tags     = ['rocm']
 
-    maintainers = ['arjun-raj-kuppala', 'srekolam']
+    maintainers = ['renjithravindrankannath', 'srekolam']
```

Comment:
please add cgmb as additional maintainer.

Reply:
Added cgmb ad maintainer in hipfft